spec/spec_helper.rb

```diff
@@ -13,6 +13,7 @@
 require "carrierwave/base64"
 
 ActiveRecord::Base.establish_connection adapter: "sqlite3", database: ":memory:"
+#ActiveRecord::Base.establish_connection adapter: "postgresql", database: "carrierwave-base64-test", username: "USERNAME", password: "YOURPASSWD"
```

Comment:
Do you consider deleting this line?

https://medium.com/@kentcdodds/please-don-t-commit-commented-out-code-53d0b5b26d5f#.g49vzgtmd